src/glmtools.jl

```diff
@@ -58,11 +87,34 @@ glmvar(::Gamma, ::Link, μ, η) = abs2(μ)
 glmvar(::Normal, ::Link, μ, η) = 1
 glmvar(::Poisson, ::Link, μ, η) = μ
 
+"""
+    mustart(d, y, wt)
+
+Derive starting values for the mu vector.
+
+#### Arguments:                                                                             
+* `d`: Distribution, can be any one of `::Binomial`, `::Gamma`, `::Normal` or `::Poisson`  * `y`:                                                                                     * `wt`:
```

Comment:
Same remark as above regarding weights. `d` must be e.g. `Binomial()`, not `::Binomial`. Also list ` Bernouilli()` after rebasing.